docs/src/catalyst_for_new_julia_users/intro_to_catalyst.md

```diff
@@ -0,0 +1,182 @@
+
+We thus activate our three desired packages:
+
+```julia
+using Catalyst
+using DifferentialEquations
+using Plots
+```
+
+For a more detailed introduction to Julia packages, please read [the Pkg documentation](https://docs.julialang.org/en/v1/stdlib/Pkg/).
+
+### Simulating a basic Catalyst model
+Now that we have some basic familiarity with Julia, and have installed and activated the required packages, we will create and simulate a basic chemical reaction network model through Catalyst.
+
+Catalyst models are created through the `@reaction_network` *macro*. For more information on macros, please read [the Julia documentation on macros](https://docs.julialang.org/en/v1/manual/metaprogramming/#man-macros). This documentation is, however, rather advanced (and not required to use Catalyst). We instead recommend that you simply familiarise yourself with the Catalyst syntax, without studying in detail how macros work and what they are.
+
+The `@reaction_network` command is followed by the `begin` keyword, which is followed by one line for each *reaction* of the model. Each reaction consists of a *reaction rate*, followed by the reaction itself. The reaction itself contains a set of *substrates* and a set of *products* (what is consumed and produced by the reaction, respectively). These are separated by a `-->` arrow. Finally, the model ends with the `end` keyword, followed by a list of its *parameters*.
+
+Here, we create a simple *birth-death* model, where a single species (*X*) is created at rate *b*, and degraded at rate *d*. The model is stored in the variable `rn`.
+
+```julia
+rn = @reaction_network begin
+    b, 0 --> X
+    d, X --> 0
+end b d;
+```
+
+For more information on how to use the Catalyst model creator, please read [the corresponding documentation](https://docs.sciml.ai/Catalyst/stable/tutorials/dsl/).
+
+Next, we wish to simulate our model. To do this, we need to provide some additional information to the simulator. This is
+* The initial condition. That is the concentration of each species at the start of the simulation.
+* The timespan. That is, over the timeframe over which we wish to run the simulation.
+* The parameter values. That is, the values of the model's parameters for this simulation.
+
+The initial condition is given as a *Vector*. This is a type which combines several different values. When declared, the values are contained by `[]`, and separated by `,`. Since we only have one species, the vector holds a single element. In this element, we set the value of *X* using the `:X => 1.0` syntax. Here, we first denote the name of the species, with a `:` pre-appended. Next follows `=>` and then the value of *X*. Since we wish to simulate the *concentration* of X over time, we will let the initial condition be decimal valued.
```

Comment:
```suggestion
The initial condition is given as a *Vector*. This is a type which collects several different values. To declare a vector, the values are specific within brackets, `[]`, and separated by `,`. Since we only have one species, the vector holds a single element. In this element, we set the value of *X* using the `:X => 1.0` syntax. Here, we first denote the name of the species, with a `:` pre-appended. Next follows `=>` and then the value of *X*. Since we wish to simulate the *concentration* of X over time, we will let the initial condition be decimal valued.
```